frontend/test/pytest/test_measurements_results.py

```diff
@@ -627,7 +627,7 @@ def test_multiple_return_values(self, backend):
         """Test multiple return values."""
 
         @qjit
-        @qml.qnode(qml.device(backend, wires=2, shots=100))
+        @qml.qnode(qml.device(backend, wires=2, shots=10000))
```

Comment:
Why so many shots? 😅 (not sure if it impacts the runtime or not)

Reply:
Fixed! :smile: